packages/testlab/src/testlab.ts

```diff
@@ -6,14 +6,17 @@
 /// <reference path="../should-as-function.d.ts" />
 
 const shouldAsFunction: Internal = require('should/as-function');
+require('should-sinon');
```

Comment:
Unless I'm mistaken, you should be able to write this as `import 'should-sinon';` and it will do all the same global importing.